### ISSUE TITLE
Letter 'e' has rendering problem for black at 27px on zdnet.com

### ISSUE THREAD

**Issue by SergeyMalkin** [impallari/Raleway#26]
I can't pinpoint particular version in this repo, but WOFF binary on zdnet.com has a problem. Maybe you can find same error in one of the files here. Curiously, ttf version on zdnet.com is different and doesn't have this problem.

![image](https://user-images.githubusercontent.com/15622197/36045200-c90bff82-0d89-11e8-88c9-d877131ce19b.png)

Direct urls to fonts on zdnet: 

http://www.zdnet.com/bundles/zdnetcss/fonts/raleway/Raleway-Black.woff
http://www.zdnet.com/bundles/zdnetcss/fonts/raleway/Raleway-Black.ttf

**Comment by davelab6** [comment]
Thanks @SergeyMalkin :) 

I am curious, are you able to reproduce this anywhere else, like Google Fonts?

cc @m4rc1e

**Comment by SergeyMalkin** [comment]
No, only with this particular version from zdnet and only woff flavor of it. It may not be even vanilla version from GitHub or GF.
